ops-867 Update data acknowledgment SOP (#1027)
ebi-ait/hca-ebi-wrangler-central: docs/SOPs/dataset_acknowledgement_SOP.md

1) Go to the `scripts/populate_ingest/` folder inside your cloned version of the `hca-ebi-dev-team` repository

2) Create a `.env` file in `scripts/populate_ingest/`. Specify the relevant ingest api url and ingest api token in
this file. For  instructions on how to obtain the token see [this guide](https://ebi-ait.github.io/hca-ebi-dev-team/operations_tasks/api_token.html).
<br>To run against dev:

    ```
        INGEST_API_URL= https://api.ingest.dev.archive.data.humancellatlas.org

>         INGEST_API_TOKEN="BearerToken <dev authentication token>"

Reviewer: ```suggestion
        INGEST_API_TOKEN="Bearer <dev authentication token>"
```

    ```
   To run against **prod**:

    ```
        INGEST_API_URL= https://api.ingest.archive.data.humancellatlas.org

>         INGEST_API_TOKEN="BearerToken <prod authentication token>"

Reviewer: ```suggestion
        INGEST_API_TOKEN="Bearer <prod authentication token>"
```

    ```

>    By default, the script runs against local host

Reviewer: ```suggestion
   By default, the script runs in dry-run mode, avoiding API operations
```
